superset/databases/filters.py

```diff
@@ -51,3 +57,38 @@ def apply(self, query: Query, value: Any) -> Query:
                 ),
             )
         )
+
+
+class DatabaseUploadEnabledFilter(BaseFilter):  # pylint: disable=too-few-public-methods
+    """
+    Custom filter for the GET list that filters all certified charts
+    """
+
+    name = _("Upload Enabled")
+    arg_name = "upload_is_enabled"
+    model = Database
+
+    def apply(self, query: Query, value: any) -> Query:
+        extra_allowed_databases = []
```

Comment:
since we don't use this again, you can remove it and just return 
```
pp.config["ALLOWED_USER_CSV_SCHEMA_FUNC"](
                Database, g.user
            )
            ```
            on line 74